app/models/user.rb

```diff
@@ -155,6 +155,7 @@ def current_student?
 
   def self.search(search)
     where("users.name ILIKE ?", "%#{search}%")
+    where("teams.name ILIKE ?", "%#{search}%")
```

Comment:
Hey @F3PiX, I think chaining `where` statements constructs `AND` queries in SQL. Furthermore, I believe the teams association has to be added with `includes(:teams).references(:teams)`, too.